build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
Was there a particular reason that you changed the build dependencies for the project? It worked fine with 2.2.3.